starlark/unpack.go

```diff
@@ -172,6 +182,11 @@ func unpackOneArg(v Value, ptr interface{}) error {
 		}
 		*ptr = it
 	default:
+		unpacker, ok := ptr.(Unpacker)
+		if ok {
```

Comment:
This could be a switch case, so let's do that:
```
case Unpacker:
   return ptr.Unpack(v)
```

Please insert it as the first case. Order doesn't matter, since the cases are all currently disjoint, but if we add new cases I think this one should take priority.

Sorry for not pointing this out before.

Reply:
good point, done